Dockerfile

```diff
@@ -13,19 +13,28 @@ RUN dnf install -y gcc git make xz-devel
 WORKDIR /tmp
 
 RUN git clone https://github.com/ipxe/ipxe.git && \
-      cd ipxe && \
-      git reset --hard $IPXE_COMMIT_HASH && \
-      cd src && \
-      ARCH=$(uname -m | sed 's/aarch/arm/') && \
-      # NOTE(elfosardo): warning should not be treated as errors by default
-      NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
+     cd ipxe && \
+     git reset --hard $IPXE_COMMIT_HASH && \
+     cd src && \
+     ARCH=$(uname -m | sed 's/aarch/arm/') && \
+     # NOTE(elfosardo): warning should not be treated as errors by default
+     NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
 
 COPY prepare-efi.sh /bin/
 RUN prepare-efi.sh centos
 
+# build actual image
 FROM $BASE_IMAGE
 
-ENV PKGS_LIST=main-packages-list.txt
+LABEL org.opencontainers.image.description="Container image to run OpenStack Ironic as part of Metal³"
+LABEL org.opencontainers.image.documentation="https://book.metal3.io/ironic/introduction"
+LABEL org.opencontainers.image.licenses="Apache License 2.0"
+LABEL org.opencontainers.image.title="Metal3 Ironic Container"
+LABEL org.opencontainers.image.url="https://github.com/metal3-io/ironic-image"
+LABEL org.opencontainers.image.vendor="Metal3-io"
+LABEL org.opencontainers.image.version="latest"
```

Comment:
When taking in release automation, we want to update this as part of the build, instead of committing a change to repo. Need to check if the image builder action supports this.

If it doesn't, I rather leave this off, than update this always in a PR before cutting a release.

Reply:
Both docker and podman support adding this with `docker build --label ...`. Hopefully we can do the same in the automated builds.
Regarding the CS9 and CS10, how do we handle that currently? Do we have different tags for them? I'm thinking that the version label could be the same as the image tag.

Reply:
We have two different images in the image-build-action.yml workflow: `ironic`  for CS9 and `ironic_cs10` for CS10 images.

Reply:
https://github.com/marketplace/actions/docker-build-push-action#inputs supports labels.

I think we should thus leave it off the Dockerfile, and set it during build with 
1. primarily tag if build trigger was tagged (release build)
2. secondarily with the branch name; if branch is main, then latest (push triggered build)

WDYT?

Reply:
+1, sounds good!

Reply:
sounds good to me